addons/account_edi/models/account_edi_format.py

```diff
@@ -379,3 +380,88 @@ def _update_invoice_from_attachment(self, attachment, invoice):
                         res.write({'extract_state': 'done'})
                     return res
         return self.env['account.move']
+
+    ####################################################
+    # Import helpers
+    ####################################################
+
+    def _retrieve_partner(self, name=None, phone=None, mail=None, vat=None, company=None):
+        '''Search all partners and find one that matches one of the parameters.
```

Comment:
According PEP257, docstring should have double quotes for consistency instead of simple quotes:

https://www.python.org/dev/peps/pep-0257/

Reply:
We don't respect PEP257 in Odoo :/